packages/input_secrets/src/input_secrets.ts

```diff
@@ -69,7 +69,11 @@ export function decryptInputSecrets<T>(
             const match = value.match(ENCRYPTED_INPUT_VALUE_REGEXP);
             if (!match) continue;
             const [, encryptedPassword, encryptedValue] = match;
-            decryptedInput[key] = privateDecrypt({ privateKey, encryptedPassword, encryptedValue });
+            try {
+                decryptedInput[key] = privateDecrypt({ privateKey, encryptedPassword, encryptedValue });
+            } catch {
+                throw new Error(`The input field "${key}" could not be decrypted. You can fix this by updating the field's value in the input editor.`);
```

Comment:
To get around the swallowing of the original error, maybe we could add the original error as `cause` to the new one, so it would be accessible?

And for the message I'd go with something like this - not saying it fixes it, just that it might, and giving some more details.

`The input field "${key}" could not be decrypted. Try updating the field's value in the input editor. Decryption error: ${err}`